sos/plugins/lxd.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2016 Jorge Niedbalski <niedbalski@ubuntu.com>
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, UbuntuPlugin
+
+
+class LXD(Plugin, UbuntuPlugin):
+    """LXD is a containers hypervisor.
+    """
+    plugin_name = 'lxd'
+
+    def setup(self):
+        self.add_copy_spec([
+            "/var/log/lxd",
```

Comment:
Please add size limiting for text log files in `/var/log` so that the plugin obeys the global `--log-size` option - there are a bunch of plugins you can use as examples (just grep for "log_size" in the plugins directory).